npf/types/web/web.py

```diff
@@ -0,0 +1,30 @@
+import uuid
+
+from npf_web_extension import app
+
+def prepare_web_export(datasets, all_results_df, path):
+
+    # Getting parameters and measurements
+    name = "undefined"
+    parameters = []
+    measurements = []
+    for testie, _, runs in datasets:

```

Comment:
For the same reasons than https://github.com/kohler/click/pull/468 where the name "testie" is inspired from, could you change references of testie to "test"? or npf test if appropriate.